tests/vm_test.go

```diff
@@ -634,6 +634,31 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			}
 		})
 
+		It("should set the Ready condition on VM", func() {
+			vm := newVirtualMachine(false)
+
+			getVmReadyConditionStatus := func() *k8sv1.ConditionStatus {
```

Comment:
Same here... why can't we use `getCondition` and then check the Status?

Reply:
Done